js/performance/demos/meded90/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Измерение времени работы иерархий медленных  функций – Performance – Doka</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    body {
+      margin: 8px 16px;
+      font-family: "Roboto", sans-serif;
+      font-size: 18px;
+      font-weight: 400;
+      background-color: #18191c!important;
+      color: #FFFFFF;
+    }
+    ol{
+      margin-bottom: 16px;
+      padding-left: 20px;
+    }
+    li{
+      margin-bottom: 6px;
+    }
+
+    .result {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .button {
+      padding: 4px 8px;
+    }
+
+    .hidden {
+      display: none;
+    }
+  </style>
+</head>
+<body>
+
+
+    <main>
+    <div id="container">
+      <ol>
+        <li> Откройте devtools -> Performance  </li>
+        <li> Кликтие на круглу  кнопку  для начла записи или <code>ctrl+e</code></li>
```

Comment:
Наш стайлгайд велит писать через пробел, а не через плюс :)

```suggestion
        <li>Нажмите круглую кнопку для начала записи или <code>Ctrl E</code></li>
```